data/json/recipes/recipe_others.json

```diff
@@ -3054,18 +3059,34 @@
     "qualities": [ { "id": "DIG", "level": 2 } ],
     "components": [ [ [ "bag_canvas", 1 ] ], [ [ "material_gravel", 2700 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "ACTIVE_EXERCISE",
+    "result": "anvil_crude",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "20 h",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "GRIND", "level": 2 }, { "id": "ANVIL", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ], [ "casting_mold", -1 ] ], [ [ "forge", 200 ], [ "oxy_torch", 200 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_blacksmithing" } ],
+    "components": [ [ [ "steel_lump", 28 ] ] ]
+  },
   {
     "type": "recipe",
     "activity_level": "MODERATE_EXERCISE",
     "result": "anvil_crude",
+    "id_suffix": "angle_grind",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
     "difficulty": 3,
     "time": "4 h",
     "autolearn": true,
-    "qualities": [ { "id": "HAMMER", "level": 2 } ],
-    "tools": [ [ [ "angle_grinder", 200 ] ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "ANVIL", "level": 1 } ],
+    "tools": [ [ [ "angle_grinder", 200 ] ], [ [ "forge", 100 ], [ "oxy_torch", 100 ] ] ],
```

Comment:
An oxy torch is fine as an alternate for cutting the rail,  but what's the forge for? Hardening? One of the reasons to use rails is they're already sufficiently hard.

Reply:
To soften enough to bend, was my thinking. Most videos I've seen using rail stock cut the stock, heat it with a blowtorch or forge, and then hammer it into a broader shape. Same reason I added a bit of anvil in there. The Hammer was already included, so I assumed they were following that method.

Trying to shape hardened steel with a hammer without heating it would risk fracturing it.

Reply:
On more research on RR anvils, it also looks like most people strongly recommend adding material to the rail support using hammer-welding. Though you can also apparently partly encase the anvil in sand or dirt, so it probably would work ok with a little work. Also the rail anvil 'works' even without these methods, it just is harder to use.